sections/performance/block-loop.spanish.md

```diff
@@ -0,0 +1,50 @@
+# No bloquees el event loop
+
+<br/><br/>
+
+Node maneja el Event Loop mayormente en un único hilo rotando a través de múltiples colas. Operaciones con alta complejidad, parseo de largos json, aplicar lógica sobre grandes arrays, consultas inseguras de regex, y largas operaciones de IO son algunas de las operaciones que pueden causar que el Event Loop se sobrecargue. Evitar esta descarga de tareas intensas de la CPU a un servicio dedicado (por ejemplo, un job server), o separar largas tareas en pequeños pasos y luego usar el Worker Pool son algunos ejemplos de como evitar bloquear el Event Loop.
+
+### Ejemplo: bloqueando el event loop
+Observemos un ejemplo de [Node Clinic](https://clinicjs.org/documentation/doctor/05-fixing-event-loop-problem).
+```
```

Comment:
We can add a `javascript` language specifier here to get syntax coloring

Reply:
In order to be as the one at the master branch, it should have the language specifier